### ISSUE TITLE
Missing event filters

### ISSUE THREAD

**Issue by stathissideris** [cljfx/cljfx#2]
This library doesn't seem to support event filters, which can intercept an event during the event capturing phase (which is when it's being propagated from root to target). See:

https://docs.oracle.com/javafx/2/events/processing.htm

**Comment by vlaaad** [comment]
You are right, I'll think how to add them.

**Comment by vlaaad** [comment]
While cljfx does not support event filters declaratively, I just released 1.1.0 that allows to add missing functionality using [extension lifecycles](https://github.com/cljfx/cljfx#included-extension-lifecycles):
```clj
{:fx/type fx/ext-on-instance-lifecycle
 :on-created (fn [^Node node]
               (.addEventFilter node Event/ANY (reify EventHandler
                                                 (handle [_ e]
                                                   (.consume e)))))
 :desc {:fx/type :v-box
        :children [,,,]}}
```
